redaxo/src/core/pages/system.settings.php

```diff
@@ -276,29 +268,29 @@
                     ))->render() ?>
 
                     <?= $viaCookie ? (new Info(
-                        slotDefault: new Slot(rex_i18n::msg('system_editor_note_cookie')),
+                        body: rex_i18n::msg('system_editor_note_cookie'),
                     ))->render() : '' ?>
                 <?php }),
 
-                slotFooter: new Slot(static function () use ($viaCookie) { ?>
+                footer: new Html(static function () use ($viaCookie) { ?>
                     <div>
                         <?= (new Button\Save(
-                            slotDefault: new Slot(rex_i18n::msg('system_editor_update_cookie')),
+                            label: rex_i18n::rawMsg('system_editor_update_cookie'),
```

Comment:
Teils müssen aktuell die i18n-Meldungen raw übergeben werden, um Doppel-Escaping zu vermeiden.
Für `rex_i18n` bin ich noch am Grübeln über ein insgesamt besseres Konzept. Ist aber hier out of scope.